RecoTracker/IterativeTracking/python/HighPtTripletStep_cff.py

```diff
@@ -150,10 +158,12 @@
 )
 trackingPhase1PU70.toModify(highPtTripletStepTrajectoryBuilder,
     MeasurementTrackerName = '',
-    maxCand = 4,
```

Comment:
Doesn't matter, `trackingPhase1PU70` is not used in any workflow, and I'll clean it up sometime soon.